app/templates/components/collapsed-objectives-program-years.hbs

```diff
@@ -1,3 +1,4 @@
+{{! template-lint-disable no-bare-strings }}
```

Comment:
Was this breaking on `&nbsp;`?

Reply:
all of those are breaking on HTML entities. some of those are non-breaking white-spaces.